src/components/basecamp/OrganizationForm.tsx

```diff
@@ -0,0 +1,263 @@
+ * @param s input string
+ * @returns
+ */
+const isMuuidHexStr = (s: string): boolean => {
+  const regex = /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/
+  return regex.test(s)
+}
+
+const MUUID_VALIDATION = {
+  validate: (value: string) => {
+    return conjoinedStringToArray(value).every(isMuuidHexStr) || 'Expected comma-separated MUUID hex strings eg. 49017dad-7baf-5fde-8078-f3a4b1230bbb, 88352d11-eb85-5fde-8078-889bb1230b11...'
+  }
+}
+
+interface HtmlFormProps extends OrganizationEditableFieldsType {
+  conjoinedAssociatedAreaIds: string // Form will return one large conjoined string
+  conjoinedExcludedAreaIds: string // Form will return one large conjoined string
+  orgId: string
+  orgType: OrgType
+}
+
+interface OrganizationFormProps {
+  existingOrg: OrganizationType | null
+  onClose: () => void
+}
+
+/*
+ * Form for creating and updating organzations.
+ * When existingOrg is null, form creates new org.
+ * When not null, form updates the existingOrg instead.
+*/
+export default function OrganizationForm ({ existingOrg, onClose }: OrganizationFormProps): JSX.Element {
+  const session = useSession()
+  const [addOrganization] = useMutation<{ addOrganization: OrganizationType }, { input: AddOrganizationProps }>(
+    MUTATION_ADD_ORGANIZATION, {
+      client: graphqlClient,
+      onError: (error) => toast.error(`Unexpected error: ${error.message}`),
+      refetchQueries: [{ query: QUERY_ORGANIZATIONS }]
+    }
+  )
+  const [updateOrganization] = useMutation<{ updateOrganization: OrganizationType }, { input: UpdateOrganizationProps }>(
+    MUTATION_UPDATE_ORGANIZATION, {
+      client: graphqlClient,
+      onError: (error) => toast.error(`Unexpected error: ${error.message}`),
+      refetchQueries: [{ query: QUERY_ORGANIZATIONS }]
+    }
+  )
+  // React-hook-form declaration
+  const form = useForm<HtmlFormProps>({
+    mode: 'onBlur',
+    defaultValues: existingOrg == null
+      ? {}
+      : {
+          displayName: existingOrg.displayName,
+          orgType: existingOrg.orgType,
+          conjoinedAssociatedAreaIds: existingOrg.associatedAreaIds?.join(', '),
+          conjoinedExcludedAreaIds: existingOrg.excludedAreaIds?.join(', '),
+          description: existingOrg.content?.description,
+          website: existingOrg.content?.website,
+          email: existingOrg.content?.email,
+          instagramLink: existingOrg.content?.instagramLink,
+          donationLink: existingOrg.content?.donationLink,
+          facebookLink: existingOrg.content?.facebookLink
+        }
+  })
+
+  const { handleSubmit, formState: { isSubmitting } } = form
+
+  /**
+   * Routes to addOrganization or updateOrganization GraphQL calls
+   * based on whether there was an existing org to update or not.
+   * @param formProps Data populated by the user in the form
+   * @returns
+   */
+  const submitHandler = async (formProps: HtmlFormProps): Promise<void> => {
+    if (existingOrg === null) {
```

Comment:
I'd use 2 equal signs, to catch both nullness and undefined check.  If react-hook-form returns `undefined` then existingOrg is not null.

`if (existingOrg == null) {`